lensterxyz/lenster
# fix: speed up conversation list

## What does this PR do?
This PR is a WIP to debug conversation list latency. I was able to reproduce 10s + loading times with a wallet that has 51 conversations in it.

| Before (main) | After (ea/fix-conversations-lock) |
|--|--|
| <img width="826" alt="Screenshot 2022-12-06 at 7 26 33 PM" src="https://user-images.githubusercontent.com/556051/206056701-85693565-3ed1-43e2-9928-55676004bfcb.png"> | <img width="825" alt="Screenshot 2022-12-06 at 6 55 12 PM" src="https://user-images.githubusercontent.com/556051/206054033-1dcc62e5-6d1d-461d-b5df-9972a1c46d85.png"> |

## Type of change


## How should this be tested?

- Use the preview link to load `/messages` page and ensure it loads faster than on lenster.xyz (especially @bigint who is very popular!)
